api/tests/config.py

```diff
@@ -0,0 +1,11 @@
+REQUEST_URL = 'http://127.0.0.1'
+REQUEST_PORT = 8080
+REQUEST_ROUTE = '/inference'
+
+SUCCESS_CODE = 200
+
+IMAGE_PATH = '../app/demo/demo_img1.jpg'
```

Comment:
This does not belong in api's config.

Reply:
same above
this `config.py` is only used in pytest for testing.